.github/workflows/ci.yml

```diff
@@ -189,21 +193,36 @@ jobs:
           # setting up maven often times out on macOS
           maven: ${{ matrix.os != 'macos-latest' }}
       - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
         with:
           go: false
-          maven-extra-args: -am -pl qa/integration-tests -T1C
-      - name: Run smoke test
+          maven-extra-args: -T1C
+      - uses: ./.github/actions/build-docker
+        id: build-docker
+        # Currently only Linux runners support building docker images without further ado
+        if: ${{ runner.os == 'Linux' }}
+        with:
+          version: current-test
+          distball: ${{ steps.build-zeebe.outputs.distball }}
+          platforms: linux/${{ matrix.arch }}
+          push: false
+      - name: Run smoke test on ${{ matrix.arch }}
+        env:
+          DOCKER_DEFAULT_PLATFORM: linux/${{ matrix.arch }}
+          # For non Linux runners there is no container available for testing, see build-docker job
+          EXCLUDED_TEST_GROUPS: ${{ runner.os != 'Linux' && 'container' }}
         run: >
           mvn -B --no-snapshot-updates
           -DskipUTs -DskipChecks -Dsurefire.rerunFailingTestsCount=3
           -pl qa/integration-tests
           -P smoke-test,extract-flaky-tests
+          -Dsmoke.test.excludedGroups=$EXCLUDED_TEST_GROUPS
```

Comment:
:wrench: Surefire [already has a property for this](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#excludedGroups), e.g. `excludedGroups`. So if you just set it to `-DexcludedGroups` then you don't need to define an additional property. I guess it wouldn't play nice with profiles which may set additional excluded groups? I'm not 100% sure there, but it shouldn't be an issue here specifically.

Reply:
good point!
will actually go for the already available user property, especially for this particular case "don't run container tests" I guess it would be nice if it also applies to different profiles, if we reuse that tag at some point.